api/src/pcapi/routes/native/v1/serialization/offers.py

```diff
@@ -270,7 +270,7 @@ def get(self, key: str, default: Any = None) -> Any:
 
         if key == "extraData":
             raw_extra_data = product.extraData if product else offer.extraData or {}
-            extra_data = OfferExtraDataResponse.parse_obj(offer.extraData)
+            extra_data = OfferExtraDataResponse.parse_obj(raw_extra_data)
```

Comment:
```suggestion
            raw_extra_data = product.extraData if product else offer.extraData
            extra_data = OfferExtraDataResponse.parse_obj(raw_extra_data or {})
```
On veut bien prendre les `extraData` du produit si elles existent non ?

Reply:
Pour moi les deux versions sont identique mais ta proposition est peut-être plus idiomatique python. Je prends

Reply:
Ah mais non ça marche pas, plus loin on fait un `raw_extra_data.get(...)` si raw_extra_data est `None` dans ce cas on a un promblème

Reply:
Ah j'avais mal lu la PR et le problème en fait. Autant pour moi 😅 
Mais pour le coup il faut quand même parer le cas où `product.extraData` vaut `None`.

Reply:
```
            raw_extra_data = (product.extraData if product else offer.extraData) or {}
```

Reply:
Yes en effet, en cherchant un peu, le cas n'existe pas mais peut-être qu'un jour ça sera le cas